config/currency_iso.json

```diff
@@ -2594,16 +2594,16 @@
     "priority": 100,
     "iso_code": "ZMW",
     "name": "Zambian Kwacha",
-    "symbol": "ZK",
+    "symbol": "K",
     "disambiguate_symbol": "ZMW",
     "alternate_symbols": [],
     "subunit": "Ngwee",
     "subunit_to_unit": 100,
-    "symbol_first": false,
+    "symbol_first": true,
     "html_entity": "",
     "decimal_mark": ".",
     "thousands_separator": ",",
     "iso_numeric": "967",
-    "smallest_denomination": 5
+    "smallest_denomination": 2
```

Comment:
From what I can tell 5 ngwee is still the smaller denomination https://en.wikipedia.org/wiki/Zambian_kwacha. Am I missing something?

Reply:
Ah, my bad. Assumed that `smallest_denomination` was a value in the currency, not in the subunit currency.

But it makes more sense to store the smallest denomination of the subunit currency. Will push up an update.

Reply:
@antstorm see latest commit to resolve your comment